### ISSUE TITLE
[a11y]: Agencies View - accessibility findings

### ISSUE THREAD

**Issue by lsr-explore** [usdigitalresponse/usdr-gost#3899]
### Why is this issue important?

This ticket contains multiple accessibility issues in arpa_reporter.  As support for Arpa reporter is winding down we collected issues in one page.  

Note:  These were identified during a developer accessibility review to double check the accessibility of the elements on the page.   To fully insure the site is accessible to users, more comprehensive testing by an accessibility subject matter expert should be performed.

Testing approach
All testing was done against [staging](https://staging.grants.usdr.dev/arpa_reporter) and local dev.
1) Run [axe DevTools chrome extension ](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)against the page.   This checks contrast 
2) Run through the page with the keyboard - I used Microsoft's [Accessibility Insights Chrome Extension](https://chromewebstore.google.com/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni) to track keyboard focus
3) Zoom the page to 200x and 400x to check reflow
4) Run through the page using a screen reader - mac voice over
5) Validate the HTML by pasting parts of the html into [W3C Markup Validation Service](https://validator.w3.org/#validate_by_input)

Testing gaps
1) Windows screen readers - JAWS / NVDA
2) Mobile web devices - the arpa reporter was not designed to be responsive
3) Real user workflow steps.   

----
# Agencies View

## axe DevTools report is clean

![Image](https://github.com/user-attachments/assets/78abae60-9f0a-48a2-b079-4725be56dca9)

Create new agency
![Image](https://github.com/user-attachments/assets/94a461eb-922e-4ef6-a629-4128214f6ddf)

Edit agency
![Image](https://github.com/user-attachments/assets/37d17e5c-781e-485b-ad30-87a069820588)

----

## issue 1 - Edit buttons should have more specific aria-label

![Image](https://github.com/user-attachments/assets/4eea0697-2df7-40ba-a2c9-05c917ffb800)

### Current State

See issues above

### Expected State

No a11y issues

### Implementation Plan

TBD

### Relevant Code Snippets

```shell

```
